StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -103,6 +105,15 @@ Scene* LeosTestScene(SceneManager* sm)
     avc->Play(L"Music");
     ic->Init();
 
+    // Skybox
+    Texture* skyboxCubemap = al->LoadTextureCubeMap(L"../Vendor/Resources/Textures/CubeMaps/skymap.dds");
+    entity = scene->AddEntity("skybox");
+    component::SkyboxComponent* sbc = entity->AddComponent<component::SkyboxComponent>();
+    sbc->SetMesh(cubeModel->GetMeshAt(0));
```

Comment:
Will sbc ever have more then one mesh? else maybe fix this

Reply:
Yes, for simplicity the skybox should only have one mesh, not a model.